constructor.js

```diff
@@ -0,0 +1,147 @@
+var allMonths = [];
+
+var january = new Month('january');
+var february = new Month('february');
+var march = new Month('march');
+var april = new Month('april');
+var may = new Month('may');
+var june = new Month('june');
+var july = new Month('july');
+var august = new Month('august');
+var september = new Month('september');
+var october = new Month('october');
+var november = new Month('november');
+var december = new Month('december');
+
+function startUpCheckStorage() {
+
+  var newBill = [];
+  if (localStorage.storedBills) {
+    var parsedBills = JSON.parse(localStorage.getItem('storedBills'));
+    for (var i = 0; i < parsedBills.length; i++){
+			var currentBill = new BillObject(parsedBills[i].billMonthName, parsedBills[i].billAmount, parsedBills[i].billFrequency, parsedBills[i].billName);
+			newBill.push(currentBill);
+			// new BillObject(each.billMonthName, each.billAmount, each.billFrequency, each.billName)
```

Comment:
Most of this commented code should be removed from the final state of the project. This goes for console.logs, which are debugging-based, and for the dev only.